src/ResourceManager/Sql/Commands.Sql/Database Backup/Services/AzureSqlDatabaseBackupCommunicator.cs

```diff
@@ -452,6 +482,21 @@ private Management.Sql.SqlManagementClient GetCurrentSqlClient()
             return SqlClient;
         }
 
+        /// <summary>
+        /// Retrieve the SQL Management client for the currently selected subscription, adding the session and request
+        /// id tracing headers for the current cmdlet invocation.
+        /// </summary>
+        /// <returns>The SQL Management client for the currently selected subscription.</returns>
+        private Management.Sql.SqlManagementClient GetCurrentArmSqlClient()
+        {
+            // Get the SQL management client for the current subscription
+            if (SqlArmClient == null)
+            {
+                SqlArmClient = AzureSession.Instance.ClientFactory.CreateArmClient<Management.Sql.SqlManagementClient>(Context, AzureEnvironment.Endpoint.ResourceManager);
```

Comment:
Same here - this looks like the exact same code as above.

Reply:
same reason